dojo/tools/anchore_grype/parser.py

```diff
@@ -36,19 +36,19 @@ def get_findings(self, file, test):
                 vuln_fix_versions = vulnerability['fix'].get('versions')
             vuln_cvss = vulnerability.get('cvss')
 
-            rel_id = None
             rel_datasource = None
             rel_urls = None
             rel_description = None
             rel_cvss = None
+            vulnerability_references = None
             related_vulnerabilities = item.get('relatedVulnerabilities')
             if related_vulnerabilities:
                 related_vulnerability = related_vulnerabilities[0]
-                rel_id = related_vulnerability.get('id')
                 rel_datasource = related_vulnerability.get('dataSource')
                 rel_urls = related_vulnerability.get('urls')
                 rel_description = related_vulnerability.get('description')
                 rel_cvss = related_vulnerability.get('cvss')
+            vulnerability_references = self.get_vulnerability_references(vuln_id, related_vulnerabilities)
```

Comment:
Attribute `relatedVulnerabilities` is an array. it could be a good opportunity to add every items.

Reply:
It could be done, but I wouldn't do it in this PR.